ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+    //i.safontev/homework/14-generics
+    public class Stack<T>
+    {
+        public T[] Arr { get; set; }
+        public int Top { get; set; }
+        public int Count { get; set; }
+
+        public Stack(int Count)
+        {
+            this.Count = Count;
+            Arr = new T[Count];
+        }
+
+        public void Push(T Element) 
+        {
+            int k = 1;
+            if (Top == Count)
+            {
+                T[] newArray = new T[Top + 1];
```

Comment:
here will be a bottleneck: each time you will copy the array, not the best way to implement stack